.test_durations

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
I guess new tests that do not occur in this file are just put in one of the buckets and not ignored? Is there a way to update the durations file from the main branch, via a separate job that records and updates the timings?

Reply:
They are not ignored. New tests are just assumed to have an average duration. So if the test suite changes significantly it can be good to update the file. It may also be smart to keep separate duration files for linux and windows.

 > Is there a way to update the durations file from the main branch, via a separate job that records and updates the timings?
 
 Yes. That would be easy enough. I could make separate workflow which would grab the duration files as artifacts, combine them to a single file and then make automated pull request. I know conda does something similar. Maybe it could run once a week/month. 

I will make a follow-up PR if you think it is worth it?

Reply:
Yes, sounds great and would be much appreciated!